lib/shared/permissions.js

```diff
@@ -143,14 +158,21 @@ exports.getEnvironmentCommandPrefix = (environment) => {
  * });
  */
 exports.elevateCommand = (command, options) => {
-  const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
+  const isWindows = os.platform() === 'win32';
+  const prefixedCommand = _.concat(
+    exports.getEnvironmentCommandPrefix(options.environment),
+    _.map(command, (argument) => {
+      return isWindows ? quoteString(argument) : argument;
```

Comment:
The quoting was done later on for non-windows, on the `commandJoin` call. I refactored the whole thing so that quoting happens in a single place!